disnake/state.py

```diff
@@ -1034,11 +1034,15 @@ def parse_thread_create(self, data) -> None:
             _log.debug("THREAD_CREATE referencing an unknown guild ID: %s. Discarding", guild_id)
             return
 
+        if data.get("newly_created") is None:
```

Comment:
`if "newly_created" not in data:` is probably a better way

Reply:
changed in c6a8925d12b80e3419dc85f26f16d40df57fb4e1, thanks